SwiftCompilerSources/Sources/SIL/Type.swift

```diff
@@ -214,6 +215,16 @@ public struct Type : TypeProperties, CustomStringConvertible, NoReflectionChildr
     }
     return false
   }
+
+  public func getEnumCasePayload(caseIdx: Int, function: Function) -> Type {
+    bridged.getEnumCasePayload(caseIdx, function.bridged).type
+  }
+
+  public func mapTypeOutOfContext() -> Type { bridged.mapTypeOutOfContext().type }
+
+  public static func getPrimitiveType(canType: CanonicalType, silValueCategory: ValueCategory) -> Type {
```

Comment:
You don't need this. Use `canType.loweredType(in:) instead. To make it an address type use `var addressType`.